chatmd/importers/codex.py

```
@@ -0,0 +1,190 @@
+class _ResponseItem(BaseModel):
+    type: str
+    payload: _Payload
+    model_config = ConfigDict(extra="allow")
+
+def _truncate_with_preview(text: str, attachment_name: str) -> str:
+    lines = text.splitlines()
+    if len(lines) <= PREVIEW_LINES * 2 + 1:
+        return text
+    head = lines[:PREVIEW_LINES]
+    tail = lines[-PREVIEW_LINES:]
+    preview = "\n".join(head + ["…", "", f"(Full content saved to {attachment_name})", ""] + tail)
```

Ruff (0.15.9)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)